data/Page/tips.md

```diff
@@ -0,0 +1,110 @@
+
+In an ideal world; a external USB condenser microphone is your best bet, but you will need to practice with it to ensure you’re getting optimal audio levels, it’s positioned correctly, it doesn’t accidentally pickup your speakers/other sound during your talk, etc.
+
+Many podcasting guides are very useful to read, for example: https://www.buzzsprout.com/blog/mic-technique-podcasting
+
+Dropping down from this, using a USB headset with a microphone is the next best thing.
+
+Practice by recording some audio in something like Audacity, and listen to the difference in audio quality between your laptop microphone or the positioning of the microphone.  Pay attention to the audio control panel in your OS to ensure you are positioned in front of the microphone correctly (if you are using an analog microphone).
+
+When presenting, try to isolate yourself away from any background/ambient noise (close windows and doors, away from people if possible, etc).
+
+## Cameras & lighting
+
+Some laptop webcams work pretty well (i.e. those in a MacBook); others aren’t so good.  An external USB webcam such as a Logitech C920 is a relatively affordable way to add high quality video.
+
+That said, before you do that, make sure you’re in a well lit position - it’s surprising how much difference this makes to quality.  Lighting should be in front of you (facing back towards you), with no visible lights behind you.
+
+An easy/cheap way to ensure you’re well lit is to sit facing a window: but keep in mind the time of day when you test, compared to the time of day when you are presenting.
+
+## Camera positioning
+
+Make sure you are filling the frame - you don’t want your head at the bottom of the shot with lots of empty space!
+
+If you can (especially if you have an external webcam and microphone), present standing up rather than sitting down.  May sound silly, but it will help considerably with your presence.  If you do this, ensure your webcam is level or facing down at you, though, rather than pointing up (a stand may help).
+
+If you’re feeling particularly creative, you could even consider a more professional setup with a dedicated HDMI camera and tripod, wireless lapel microphone, lighting and backdrop, but a decent external webcam and external microphone can get you most of the way there.
+
+## Internet connection
+
+If you can, a wired Internet connection is going to beat the pants of a wireless connection.  Latency/jitter/dropouts (which are more likely to occur on wifi) especially affect video streaming.
+
+Streaming at 720p will require at least 1mbit out, but over 5mbit up is preferred (i.e. a NBN 25/5 connection or higher).   When you are presenting, strong recommendation to ensure nobody else in your household is utilising upload bandwidth (i.e. peer to peer applications).
```

Comment:
Streaming at 720p ideally requires over 2mbit, with a minimum of 1mbit for OK quality.  5mbit or higher is highly recommended (i.e.a NBN 25/5 connection or higher).